server/routes/nl_interface_next.py

```diff
@@ -592,50 +310,67 @@ def data():
     flask.abort(404)
   original_query = request.args.get('q')
   context_history = request.get_json().get('contextHistory', [])
-  has_context = False
-  if context_history:
-    has_context = True
   logging.info(context_history)
+
   query = str(escape(nl_utils.remove_punctuations(original_query)))
   embeddings_build = str(escape(request.args.get('build', "combined_all")))
-  default_place = "United States"
-  res = {'place_type': '', 'place_name': '', 'place_dcid': '', 'config': {}}
+  res = {
+      'place': {
+          'dcid': '',
+          'name': '',
+          'place_type': '',
+      },
+      'config': {},
+      'context': context_history
```

Comment:
call `escape` on the context_history since that came from the request.
https://tedboy.github.io/flask/generated/flask.escape.html